packages/CodingStandard/src/Fixer/LineLength/BreakArrayListFixer.php

```diff
@@ -0,0 +1,153 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Fixer\LineLength;
+
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\Fixer\WhitespacesAwareFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\WhitespacesFixerConfig;
+use SplFileInfo;
+use Symplify\TokenRunner\Analyzer\FixerAnalyzer\IndentDetector;
+use Symplify\TokenRunner\Wrapper\FixerWrapper\ArrayWrapper;
+
+final class BreakArrayListFixer implements DefinedFixerInterface, WhitespacesAwareFixerInterface
+{
+    /**
+     * @var int
+     */
+    private const LINE_LENGTH = 120;
+
+    /**
+     * @var int[]
+     */
+    private const ARRAY_OPEN_TOKENS = [T_ARRAY, CT::T_ARRAY_SQUARE_BRACE_OPEN];
+
+    /**
+     * @var WhitespacesFixerConfig
+     */
+    private $whitespacesFixerConfig;
+
+    /**
+     * @var IndentDetector
+     */
+    private $indentDetector;
+
+    /**
+     * @var string
+     */
+    private $indentWhitespace;
+
+    /**
+     * @var string
+     */
+    private $newlineIndentWhitespace;
+
+    /**
+     * @var string
+     */
+    private $closingBracketNewlineIndentWhitespace;
+
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition('Array items should be on same/standalone line to fit line length.', [
+            new CodeSample(
+                '<?php
+$array = ["loooooooooooooooooooooooooooooooongArraaaaaaaaaaay", "looooooooooooooooooooooooooooooooongArraaaaaaaaaaay"];'
+            ),
+        ]);
+    }
+
+    public function isCandidate(Tokens $tokens): bool
+    {
+        return $tokens->isAnyTokenKindsFound(self::ARRAY_OPEN_TOKENS)
+            && $tokens->isAllTokenKindsFound(['=']);
```

Comment:
Can't we use [`isTokenKindFound`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.10/src/Tokenizer/Tokens.php#L1149-L1156) here?